<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteRuleDialog</class>
 <widget class="QDialog" name="DeleteRuleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>176</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>192</width>
    <height>176</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>328</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>CommitMono</family>
      </font>
     </property>
     <property name="text">
      <string>Select a rule to delete</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listWidget_Rules">
     <property name="font">
      <font>
       <family>CommitMono</family>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_Delete">
       <property name="font">
        <font>
         <family>CommitMono</family>
        </font>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="font">
        <font>
         <family>CommitMono</family>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="font">
        <font>
         <family>CommitMono</family>
        </font>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
